docker/prepare_proxy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
Can you add `set -e` to make sure initialization fails if iptables fails for some reason?